client/js/requests.js

```diff
@@ -0,0 +1,46 @@
+//Create event listener for entire window
+window.addEventListener('submit', createPost);
+
+async function createPost(e) {
+    e.preventDefault();
+    console.log(e.target.title.value);
+    const body = {
+        "title": e.target.title.value,
+        "author": e.target.author.value,
+        "post": e.target.post.value
+    };
+    console.log(`User input is ${body}.`);
+
+    const options = {
+        method: 'POST',
+        mode: 'cors',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(body)
+    }
+    let response = await fetch("http://localhost:3000/posts", options)
+        .then(r => r.json())
+        .then(data => getPost(data.id))
+        .catch(console.warn);
+  
+    return response;
+
+    
+}
+
+
+async function getPost(id){
```

Comment:
Maybe you could get the server to return a single post after creating instead of all the posts that way this function wouldn't need to make a fetch request